base/test/mp4writersink_tests.cpp

```diff
@@ -219,100 +266,14 @@ BOOST_AUTO_TEST_CASE(setgetprops_jpeg)
 
 BOOST_AUTO_TEST_CASE(h264_to_mp4v, *boost::unit_test::disabled())
```

Comment:
why a disabled test here ?

Reply:
There was some metadata path missing i added that metadata , now i've enabled these tests